doc/api/n-api.md

```diff
@@ -6070,6 +6070,11 @@ Node.js process exits while there is a thread-safe function still active.
 It is not necessary to call into JavaScript via `napi_make_callback()` because
 Node-API runs `call_js_cb` in a context appropriate for callbacks.
 
+Zero or more queued callbacks may be invoked in each tick of the event queue.
```

Comment:
Seems like the term "event queue" is the first appearance here. Could we replace it with a more common term?

Also, an `napi_threadsafe_function` has a single callback with multiple queued data items. I think it would be more clear to align the term as "items" as above paragraphs.

```suggestion
Zero or more queued items may be invoked in each tick of the event loop.
```

Reply:
@legendecas added your suggested changes which looked good. Agreed it should have been event loop for sure.